diff --git a/Makefile b/Makefile
index 572aff9..c535f3b 100644
--- a/Makefile
+++ b/Makefile
@@ -94,8 +94,7 @@ endif
 install.files:
 	@echo INSTALL_CAN_PUT_PERMISSIONS: ${INSTALL_CAN_PUT_PERMISSIONS}
 	 ## Command pipe
-	$(INSTALL) -m ${INSTALL_MOD_755} -d $(dir $(TEMPROOT)$(COMMANDPIPE))
-	[ -p $(TEMPROOT)$(COMMANDPIPE) ] || \
+	true || \
 	 { mkfifo -m ${INSTALL_MOD_660} $(TEMPROOT)$(COMMANDPIPE); \
 	   if [ "${INSTALL_CAN_PUT_PERMISSIONS}" = yes ]; \
 	     then $(CHOWN) -R $(OCSIGENUSER):$(OCSIGENGROUP) $(TEMPROOT)$(COMMANDPIPE); \
diff --git a/configure b/configure
index 40ddb38..386c414 100755
--- a/configure
+++ b/configure
@@ -622,7 +622,7 @@ UPLOADDIR := $root$uploaddir
 DOCDIR := $root$docdir
 
 # The name of the named pipe used to command the server
-COMMANDPIPE := $root$commandpipe
+COMMANDPIPE := /tmp/ocsigenserver_command
 
 # The source directory
 SRC := $full_pwd
